src/ReleaseApp/Application/Configuration/ReleaseAppConstant.php

```diff
@@ -34,5 +34,5 @@ class ReleaseAppConstant
     /**
      * @var string
      */
-    public const RELEASE_GROUP_LINK_PATTERN = '%s/release-groups/view/%s';
+    public const RELEASE_GROUP_LINK_PATTERN = '%s/release-groups/%s';
```

Comment:
it's wrong :) As I wrote in ticket, there should be no 's', so instead of "release-groups/" there should be "release-group/"

Reply:
Thanks ++